### ISSUE TITLE
Crashing when pressing shift and clicking the turrets or drones

### ISSUE THREAD

**Issue by XGuidedx** [AlexiyOrlov/k-turrets#15]
K-Turrets-0.1.13-1.16.5 results in a crash when holding shift and clicking upon a turret or drone. It doesn't matter which turret/drone variation, all of them crash holding shift and clicking. 

World is on a Server.
*Note: Version K-Turrets-0.1.8-1.16.5 worked perfectly before updating to the latest version available 

I enclosed 2 crash instances and the latest log for the latest crash.
[crash-2023-09-02_20.16.11-client.txt](https://github.com/AlexiyOrlov/k-turrets/files/12504993/crash-2023-09-02_20.16.11-client.txt)
[crash-2023-09-02_20.17.24-client.txt](https://github.com/AlexiyOrlov/k-turrets/files/12504994/crash-2023-09-02_20.17.24-client.txt)
[latesttosend.log](https://github.com/AlexiyOrlov/k-turrets/files/12504998/latesttosend.log)

**Comment by AlexiyOrlov** [comment]
Try using latest Satako version I released

**Comment by XGuidedx** [comment]
Hey @AlexiyOrlov,

I went back and did some testing with the new Satako version (Satako-4.2.16-1.16.5) and the result is no more crashing. Everything seems to work properly now while opening the configuration screen. 

However, I did find something else, but I'm not sure if I should open up a new ticket for the new issue found.
Basically in the configuration screen for the turret, the target list is not populating all the entities in the target list. The reason I know this is because I spawned an entity not on the list yet the turret still fired at the target(also adding a new entity does not populate the entity like in the previous version). I have screenshots to back my explanation. Just let me know if I should explain the issue further here or if should I open up a new ticket.

**Comment by AlexiyOrlov** [comment]
You can explain it here

**Comment by XGuidedx** [comment]
@AlexiyOrlov 

Okay, this is embarrassing, while I was typing my explanation, I found out that utilizing the plus and minus scrolling buttons populates the rest of the entity list.  I was used to the old alt-scroll method and thought the entities were not populating. So no issue here. I also realized that the add-on entity search bar no longer populates entities already on the list. So again no issue here. My mistake. This issue can be deemed as fixed I haven't experienced any issues in relation to the configuration menu. Thank you so much for your quick fix!!!
